image-loader/app/ImageLoaderComponents.scala

```diff
@@ -18,15 +18,20 @@ class ImageLoaderComponents(context: Context) extends GridComponents(context, ne
 
   val store = new ImageLoaderStore(config)
   val imageOperations = new ImageOperations(context.environment.rootPath.getAbsolutePath)
-
   val notifications = new Notifications(config)
   val downloader = new Downloader()
   val uploader = new Uploader(store, config, imageOperations, notifications)
-
   val projector = Projector(config, imageOperations)
-
+  val checkQuarantineFlag = config.uploadToQuarantineEnabled 
+  val quarantineUploader: Option[QuarantineUploader] = checkQuarantineFlag match {
```

Comment:
I think `checkQuarantineFlag` is superflous here. I'd put it directly inline.

```suggestion
  val quarantineUploader: Option[QuarantineUploader] = config.uploadToQuarantineEnabled match {
```